packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -444,12 +511,27 @@ class _CpuFlameChart extends CoreElement {
     stackFrameDetails.text = stackFrameDetailsDefaultText;
     stackFrameDetails.attribute('hidden', true);
 
-    error.clear();
-    error.attribute('hidden', true);
-    showingError = false;
+    _removeError();
 
     cpuProfileData = null;
   }
+
+  void _removeError() {
+    element.children.removeWhere((e) => e.id == 'flame-chart-error');
```

Comment:
nit: more standard would be
element.querySelector('#flame-char-error).remove()

Reply:
needs to be 
`element.querySelector('#flame-chart-error')` not `element.querySelector('#flame-chart-message')`

Reply:
Well `element.querySelector('#flame-chart-message')` was in addition to the changing `flame-chart-error` to `flame-chart-message`.

`element.children.removeWhere((e) => e.id == 'flame-chart-message') is not null.